include/odp/api/spec/packet_io.h

```diff
@@ -1394,6 +1397,69 @@ uint64_t odp_pktin_ts_res(odp_pktio_t pktio);
  */
 odp_time_t odp_pktin_ts_from_ns(odp_pktio_t pktio, uint64_t ns);
 
+/**
+ * Packet IO timestamp resolution in hertz
+ *
+ * This is the resolution of packet input and packet output timestamps.
+ * using Packet IO time source. Returns zero on a failure or when both
+ * packet in and packet out timestamping is disabled.
```

Comment:
typo: is ->are

Maybe also use the terms RX and TX timestamping (instead of packet in and packet out timestampig) here too, for consistency.